packages/shared/components/popups/SendConfirmationPopup.svelte

```diff
@@ -99,19 +105,9 @@
 
     async function validateAndSendOutput(): Promise<void> {
         validateSendConfirmation(outputOptions, preparedOutput)
-
-        if ($activeProfile.type === ProfileType.Ledger) {
-            setLedgerSendConfirmationProps({
-                asset,
-                amount,
-                unit,
-                recipient,
-                internal: false,
-                metadata,
-                tag,
-            })
+        if (!get(isSoftwareProfile)) {
```

Comment:
This should probably be `if ($isActiveLedgerProfile || $isOnboardingLedgerProfile) { ... }`. You probably don't need the onboarding profile since the only sending possible in all of the onboarding flows is specifically in the Shimmer claiming flow.